features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -114,6 +114,21 @@
     private final ClassificationEngine classificationEngine;
 
     private final int bulkRetryCount;
+    
+    private static String eventIndexName = "netflow";
+    
+    //public static void setEventIndexName(String eventIndexName) {
```

Comment:
The setter needs to be uncommented, otherwise the configured value won't be able to be passed into this class.

Make sure it is not static.